docs/operations/comanage-recipes.md

```diff
@@ -0,0 +1,71 @@
+COManage Recipes
+==============================
+
+A collection of step-by-step instructions for common actions for use by administrators of the OSG COManage.
+
+
+Steps for Common COManage Actions
+----
+This section contains some common actions administrators perform on the OSG COManage, and how to do so.
+
+### Provisioning a CO Group in COManage
+In order for a CO Group from COManage to show up in LDAP (and thus be made available for reference on Hosts),
+ it must first be provisioned.
+
+Follow these steps to provision a CO Group into LDAP:
+
+1. #### Create CO Group in COManage and add members (or use existing CO Group)
+
+    Skip the first bullet point if using an existing Group.
+
+    -   Navigate to the `All Groups` page in COManage and click the `+ Add Group` button near the top-right.
```

Comment:
```suggestion
    1.  **If using an already existing group, skip to the next step.** Navigate to the `All Groups` page in COManage and click the `+ Add Group` button near the top-right.
```

Combining and bolding here helps with clarity. Also, this section has two steps that should happen in order (if doing both). We use bulleted lists more when we're describing exclusive options, or an unordered list